images/blog.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Home Page</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width-device-width; initial-scale=1.0;">
+        <style>
+            
+            <link rel="stylesheet"
+                href="https://fonts.googleapis.com/css?family=Open+Sans|Raleway&display=swap"> 
+            
+            *       { box-sizing: border-box; }
+            body    { margin: 0 auto; max-width: 100%; }    
+            img     { width: 100%; }
+            h1, h2, h3, h4 { font-family: 'Open Sans', sans-serif; }
+            p       { font-family: 'Raleway', sans-serif; }
+            button  { font-family: 'Raleway', sans-serif; }
+        
+        </style>
+    
+    </head>
+    <body>
+        <header style="background-image: url('header-image.jpg')">
+            <section>
+                <!--Logo here-->
+                <img src="adoptadog-logo.svg" alt="Adoptadog">
+                <h1>Seattle's premiere dog adoption agency</h1>
+            </section>
+            <section>
+                <!--Cart here-->
+                <img src="cart-icon.svg" alt="Your Cart">
+                <p>$1234.56</p>
+            </section>
+            <button>
+                Checkout
+            </button>
+        </header>
+        <nav>
+            <ul>
+                <li> <a href="index.html">Home</a> </li>
+                <li> <p>/</p> </li>
```

Comment:
There is a way to handle these using the pseudo selectors  in css. Totally optional, but if you can figure it out, it's awesome as it's just a decorative `\`

Google the after pseudo selector and how they work (you'll need the content attribute...hint hint) If you want I'd love to go over it at the end of one of the classes, just make sure and ask and I'll go through it.